text/0059-otlp-trace-data-format.md

```diff
@@ -0,0 +1,346 @@
+  // user-supplied key-value pairs, or details of a message sent/received between Spans.
+  message TimedEvent {
+    // time_unixnano is the time the event occurred.
+    int64 time_unixnano = 1;
+
+    // name is a user-supplied description of the event.
+    string name = 2;
+
+    // attributes is a collection of attribute key/value pairs on the event.
+    repeated AttributeKeyValue attributes = 3;
+
+    // dropped_attributes_count is the number of dropped attributes. If the value is 0,
+    // then no attributes were dropped.
+    int32 dropped_attributes_count = 4;
+  }
+
+  // timed_events is a collection of TimedEvent items.
+  repeated TimedEvent timed_events = 12;
+
+  // dropped_timed_events_count is the number of dropped timed events. If the value is 0,
+  // then no events were dropped.
+  int32 dropped_timed_events_count = 13;
+
+  // Link is a pointer from the current span to another span in the same trace or in a
+  // different trace. For example, this can be used in batching operations,
+  // where a single batch handler processes multiple requests from different
+  // traces or when the handler receives a request from a different project.
+  message Link {
+    // trace_id is a unique identifier of a trace that this linked span is part of.
+    // The ID is a 16-byte array.
+    bytes trace_id = 1;
+
+    // span_id is a unique identifier for the linked span. The ID is an 8-byte array.
+    bytes span_id = 2;
+
+    // tracestate is the trace state associated with the link.
+    repeated TraceStateEntry tracestate = 3;
+
+    // attributes is a collection of attribute key/value pairs on the link.
+    repeated AttributeKeyValue attributes = 4;
+
+    // dropped_attributes_count is the number of dropped attributes. If the value is 0,
+    // then no attributes were dropped.
+    int32 dropped_attributes_count = 5;
+  }
+
+  // links is a collection of Links, which are references from this span to a span
+  // in the same or different trace.
+  repeated Link links = 14;
+
+  // dropped_links_count is the number of dropped links after the maximum size was
+  // enforced. If this value is 0, then no links were dropped.
+  int32 dropped_links_count = 15;
+
+  // status is an optional final status for this span. Semantically when status
+  // wasn't set it is means span ended without errors and assume Status.Ok (code = 0).
+  Status status = 16;
+
+  // child_span_count is an optional number of child spans that were generated while this
+  // span was active. If set, allows an implementation to detect missing child spans.
+  int32 child_span_count = 17;
+}
+
+// The `Status` type defines a logical error model that is suitable for different
+// programming environments, including REST APIs and RPC APIs.
+message Status {
+  // The status code. This is optional field. It is safe to assume 0 (OK)
+  // when not set.
+  int32 code = 1;
+
+  // A developer-facing error message, which should be in English.
```

Comment:
this is new comparing to the spec. Perhaps add a note here: https://github.com/open-telemetry/opentelemetry-specification/blob/485e6a5cd6290a9ef1eda20a6e68de897548ddc4/specification/api-tracing.md#set-status

Reply:
I mean that it should be in English

Reply:
I rephrased and removed reference to English, please have a look again.